src/attributes.jl

```diff
@@ -174,52 +174,6 @@ function get!(output, model::ModelLike, attr::AnyAttribute, args...)
     throw(ArgumentError("ModelLike of type $(typeof(model)) does not support accessing the attribute $attr"))
 end
```

Comment:
(Applies a few lines above.) The documentation for `get` needs to be updated to describe when it now returns `nothing`.